receiver/mysqlreceiver/integration_test.go

```diff
@@ -0,0 +1,122 @@
+package mysqlreceiver
```

Comment:
Needs the copyright notice. Also please add "integration" build tags.

Reply:
added